docs/build/ecosystem.md

```diff
@@ -1,59 +1,66 @@
 
 ![image alt text](../images/ecosystem.png)
 
-* **Research Libraries and Tools:** Libraries and tools used for researching new quantum algorithms and designing and preparing experiments on quantum devices.
+*   **Research Libraries and Tools:** Libraries and tools used for researching
+    new quantum algorithms and designing and preparing experiments on quantum
+    devices.
 
-* **Cirq**: A framework specifically for programming noisy intermediate-scale quantum computers.
+*   **Cirq**: A framework specifically for programming noisy intermediate-scale
+    quantum computers.
 
-* **Quantum Cloud Services:** Cirq can connect to a variety of quantum cloud services. Behind each cloud service, quantum algorithms run on either a quantum processor or simulator.
+*   **Quantum Cloud Services:** Cirq can connect to a variety of quantum cloud
+    services. Behind each cloud service, quantum algorithms run on either a
+    quantum processor or simulator.
 
-* **Quantum Circuit Simulators:** Cirq is compatible with a number of quantum circuit simulators that can run either locally or in a distributed fashion.
+*   **Quantum Circuit Simulators:** Cirq is compatible with a number of quantum
+    circuit simulators that can run either locally or in a distributed fashion.
 
 # Research libraries and tools
 
 ## Algorithm libraries and experiments
 
-|Name|Main sponsor|Description|
-|--- |--- |--- |
-|[Cirq](https://github.com/quantumlib/Cirq)|Google|An open-source framework for creating, editing, and invoking Noisy Intermediate-Scale Quantum (NISQ) circuits.|
-|[OpenFermion](https://github.com/quantumlib/OpenFermion)|Google|An open-source algorithms library for developing new quantum chemistry and materials simulation algorithms|
-|[TensorFlow Quantum](https://tensorflow.org/quantum)|Google|An open-source algorithms library for developing new quantum machine learning algorithms|
-|[Qualtran](https://github.com/quantumlib/qualtran)|Google|An open-source library for expressing and analyzing fault-tolerant quantum algorithms|
-|[Stim](https://github.com/quantumlib/stim)|Google|An open-source library for high-speed simulation of Clifford circuits and quantum error correction|
-|[ReCirq](https://github.com/quantumlib/ReCirq)|Google|An open-source repository of example experiments, tools, and tutorials in quantum computing|
-|[Forge](https://forge.qcware.com/)|QCWare|A proprietary library for domain-specific quantum algorithms and applications|
-|[Pennylane](https://pennylane.ai/)|Xanadu|An open-source library for quantum machine learning with TensorFlow, PyTorch, or NumPy|
+Name                                                     | Main sponsor | Description
+-------------------------------------------------------- | ------------ | -----------
+[Cirq](https://github.com/quantumlib/Cirq)               | Google       | An open-source framework for creating, editing, and invoking Noisy Intermediate-Scale Quantum (NISQ) circuits.
```

Comment:
Is this required by formatter / linter?
The new version takes more work to maintain as a change in column width has to be done on all lines.